here-naksha-lib-handlers/src/main/java/com/here/naksha/lib/handlers/DefaultStorageHandler.java

```diff
@@ -81,14 +81,19 @@ public DefaultStorageHandler(
     this.properties = JsonSerializable.convert(eventHandler.getProperties(), DefaultStorageHandlerProperties.class);
   }
 
-  /**
-   * The method invoked by the event-pipeline to process custom Storage specific read/write operations
-   *
-   * @param event the event to process.
-   * @return the result.
-   */
   @Override
-  public @NotNull Result processEvent(@NotNull IEvent event) {
+  protected EventProcessingStrategy processingStrategyFor(IEvent event) {
+    Request<?> request = event.getRequest();
+    if (request instanceof ReadFeatures
+        || request instanceof WriteFeatures
+        || request instanceof WriteCollections) {
+      return EventProcessingStrategy.PROCESS;
+    }
+    return EventProcessingStrategy.SUCCEED_WITHOUT_PROCESSING;
```

Comment:
I think, logically thinking for this handler (being last one before Psql operations), returning NOT_IMPLEMENTED as default will be more appropriate, else it can give false impression to consumer that request was successful and it didn't return any record.

Reply:
Done